src/components/net-neutrality/call-button.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classnames from 'classnames';
+import submit from '../../lib/submit.js';
+
+module.exports = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
+  getInitialState: function() {
+    return { submitting: false };
+  },
+  makeCall: function() {
+    if (this.state.submitting) {
+      return;
+    }
+
+    this.setState({
+      submitting: true
+    });
+
+    submit(
+      '/api/call',
+      {
+        number: this.props.number,
+        locale: this.context.intl.locale,
+        zip: "37097" // Just testing
```

Comment:
Should this be removed now?

Reply:
Yup, the zip stuff is doing locally, pushing up soon.